### ISSUE TITLE
anaconda 4.4 update + RH5 = broken anaconda

### ISSUE THREAD

**Issue by AntoineD** [conda/conda#6055]
Hello,
I update my anaconda2 4.4.0 based installation on a red hat 5.11, now it is broken because of the system GLIBC is too old:
```
/path/anaconda2-4.4.0/bin/python: /lib64/libc.so.6: version `GLIBC_2.7' not found (required by /path/anaconda2-4.4.0/bin/../lib/libpython2.7.so.1.0)
```
It looks like the update pulled new packages from anaconda 5.0.0 through a new channel which is now 
`https://repo.continuum.io/pkgs/main`
instead of
`https://repo.continuum.io/pkgs/free`
The update procedure shall check the platform requirement before making those changes.

I then tried to run conda `update --all` from a brand new anaconda2 4.4.0 installation, everything seems to be working OK, but it looks like any new package I want to install will be pulled from the new main-based default channel.

How could I stay on the old default channel so I do not break my installation?

**Comment by AntoineD** [comment]
Somehow I missed the related closed issue #6041, it has a workaround for that issue.
Still, I think conda update may check the system GLIBC version.

**Comment by kalefranz** [comment]
RHEL 5 and CentOS 5 are no longer supported in the `defaults` repositories.  End of life for security updates was March 31, 2017.

For support beyond what is provided by the `defaults` repositories, enterprise support contracts are available through Anaconda, Inc.
